### ISSUE TITLE
1.0 release doesn't appear to contain toolchains/unittest/BUILD

### ISSUE THREAD

**Issue by thomasvl** [bazelbuild/bazel-skylib#199]
1.0 release doesn't appear to contain toolchains/unittest/BUILD

```
$ wget -O - https://github.com/bazelbuild/bazel-skylib/releases/download/1.0.0/bazel-skylib-1.0.0.tar.gz 2>/dev/null | tar tzf -
./
./LICENSE
./BUILD
./CODEOWNERS
./CONTRIBUTORS
./lib/
./lib/BUILD
./lib/collections.bzl
./lib/dicts.bzl
./lib/new_sets.bzl
./lib/old_sets.bzl
./lib/partial.bzl
./lib/paths.bzl
./lib/selects.bzl
./lib/sets.bzl
./lib/shell.bzl
./lib/structs.bzl
./lib/types.bzl
./lib/unittest.bzl
./lib/versions.bzl
./rules/
./rules/BUILD
./rules/analysis_test.bzl
./rules/build_test.bzl
./rules/common_settings.bzl
./rules/copy_file.bzl
./rules/diff_test.bzl
./rules/empty_test.sh
./rules/native_binary.bzl
./rules/run_binary.bzl
./rules/write_file.bzl
./bzl_library.bzl
./internal_deps.bzl
./internal_setup.bzl
./lib.bzl
./skylark_library.bzl
./version.bzl
./workspace.bzl
```

**Comment by thomasvl** [comment]
Guess - since you manually create this, does something try to skip files/directories that match "test" and thot took out the "unittest" directory also?

**Comment by aiuto** [comment]
It's not quite manual. The packaging is to explicitly include what we want. 
https://github.com/bazelbuild/bazel-skylib/pull/201 is a fix. But that will probably not happen until monday. I put a warning up on the releases page.

**Comment by aiuto** [comment]
FWIW. I am trying to get away from manual steps for doing releases and work towards tooling that ensures people don't make copy paste errors. This will include things like uploading to the google mirror and updating url to urls

**Comment by aiuto** [comment]
The 1.0.1 preview is out.  It contains unittest.

$ tar tzf bazel-skylib-1.0.1.tar.gz 
./
./LICENSE
./BUILD
./CODEOWNERS
./CONTRIBUTORS
./lib/
./lib/BUILD
./lib/collections.bzl
./lib/dicts.bzl
./lib/new_sets.bzl
./lib/old_sets.bzl
./lib/partial.bzl
./lib/paths.bzl
./lib/selects.bzl
./lib/sets.bzl
./lib/shell.bzl
./lib/structs.bzl
./lib/types.bzl
./lib/unittest.bzl
./lib/versions.bzl
./rules/
./rules/BUILD
./rules/analysis_test.bzl
./rules/build_test.bzl
./rules/common_settings.bzl
./rules/copy_file.bzl
./rules/diff_test.bzl
./rules/empty_test.sh
./rules/native_binary.bzl
./rules/run_binary.bzl
./rules/write_file.bzl
./rules/private/
./rules/private/BUILD
./rules/private/copy_file_private.bzl
./rules/private/maprule_util.bzl
./rules/private/write_file_private.bzl
./toolchains/
./toolchains/unittest/
./toolchains/unittest/BUILD
./bzl_library.bzl
./internal_deps.bzl
./internal_setup.bzl
./lib.bzl
./skylark_library.bzl
./version.bzl
./workspace.bzl

**Comment by aiuto** [comment]
Fixed in 1.0.2
